src/command_modules/azure-cli-botservice/setup.py

```diff
@@ -36,7 +36,7 @@
 
 DEPENDENCIES = [
     'azure-cli-core',
-    'azure-mgmt-web==0.40.0',
+    'azure-mgmt-web>=0.35.0',
```

Comment:
Why? We need to pin to a version to avoid version confliction.

Reply:
Yes, please we want to avoid the same issues again the next time a new SDK version gets released.